app/views/content_items/show.html.erb

```diff
@@ -22,16 +22,23 @@
       <%= content_html %>
     </div>
     <div class="column-third">
-      <% if override_sidebar %>
-        <div class="child-topic-contents">
-          <%= render partial: "sidebar", locals: {
-            page_schema: @page_schema,
-            current_step_number: current_step_number,
-            current_task_number: current_task_number
-            } %>
-        </div>
-      <% else %>
-        <%= render partial: 'govuk_component/taxonomy_sidebar', locals: taxonomy_sidebar %>
+      <% if get_tasklist == 'divorce' %>
+        <%= render 'partials/get-a-divorce-sidebar' %>
+      <% elsif get_tasklist == 'civil' %>
+        <%= render 'partials/end-a-civil-partnership-sidebar' %>
+      <% end %>
+
+      <% if false %>
```

Comment:
what's the logic here?

Reply:
Forgot about that - it's the simplest way I've found of commenting out chunks of code in rails. This bit can be deleted.

Reply:
Hey wait, just thought - is this why the sidebar isn't working on learn to drive anymore? Not the if false I added, but one of the lines within it was removed in a prior commit. Would probably impact it.

Reply:
Maybe, I haven't got the chance to explore it yet. I'll have a look when I get a chance